### ISSUE TITLE
SNES Classic: No games will launch, not even original titles (v2.2.0)

### ISSUE THREAD

**Issue by DonkeyKong** [ClusterM/hakchi2#504]
I dumped the kernel on my SNES classic using Hakchi 2.20, then added 8 games, gave them nice metadata, and clicked sync. The app guided me through flashing a custom kernel, which I did. It then synced all my games. I closed Hakchi 2.20, clicked the power switch on the snes classic to off, waited for the light to go off, disconnected it from USB, plugged it into my TV, and turned it on.

The machine booted fine and it showed all my new games added with art and metadata. Everything looked great. Unfortunately, no games will run anymore. Not even original titles that came with the system. I click a game, the screen goes black for a second, then I'm dumped back into the main menu. 

I'm technically proficient but have little experience with Hakchi, so I can collect any debugging data needed but I'll need a little bit on instruction on how to do it.

**Comment by DonkeyKong** [comment]
Quick update with more details: I have a launch day US unit. The only setting I changed in Hakchi was I put the folder mode to Automatic, since I had 29 games total and that seemed fine.

When I flipped back to the default option, Original Games In Root, Automatic In Subfolder, I get the same problem as described in this issue (down to the last detail). https://github.com/ClusterM/hakchi2/issues/505

No games launch either way.

**Comment by ClusterM** [comment]
https://github.com/ClusterM/hakchi2/wiki/If-you-have-any-problems

**Comment by DonkeyKong** [comment]
I restored to the original firmware using the release version of hakchi2 and verified my hardware was ok.  I was able to launch and play the original games no problem. New games were gone, as expected.

I used the debug build to flash my hardware the same way I did the first time, same set of games, and had no issue at all. Everything went smoothly and now the newly added games launch.  I don't have the crash on enter folder issue anymore either.

I'll keep trying and see what I can do to reproduce the problem.
